third_party/validator/sql_database_instance.go

```diff
@@ -111,7 +111,7 @@ func expandSqlDatabaseInstanceSettings(configured []interface{}, secondGen bool)
 	// 1st Generation instances don't support the disk_autoresize parameter
 	// and it defaults to true - so we shouldn't set it if this is first gen
 	if secondGen {
-		settings.StorageAutoResize = googleapi.Bool(_settings["disk_autoresize"].(bool))
+		settings.StorageAutoResize = *googleapi.Bool(_settings["disk_autoresize"].(bool))
```

Comment:
googleapi.Bool wraps a value in a pointer, so I think you get the same effect in a simpler manner by doing:
```suggestion
		settings.StorageAutoResize = _settings["disk_autoresize"].(bool)
```